plugins/nanospell/plugin.js

```diff
@@ -452,6 +469,8 @@
 
 				editor.fire('SpellcheckStart');
 				editor.nanospellstarted = true;
+				self._isBusy = false;
+				self._timer = 0;
```

Comment:
This seems fair, it does mess with the ability to distinguish a complete timer from a never started timer, but fussing over 0 !== null is probably not a good thing to do.